tests/test_pay.py

```diff
@@ -6160,6 +6160,21 @@ def test_offer_with_private_channels_multyhop2(node_factory):
     l1.rpc.pay(invoice)
 
 
+def test_offer_with_private_channels_force_issuer_id_multyhop2(node_factory):
+    """We should be able to fetch an invoice through a private path while forcing the issuer id and pay the invoice"""
+    l1, l2, l3, l4, l5 = node_factory.line_graph(5, fundchannel=False)
+
+    node_factory.join_nodes([l1, l2], wait_for_announce=True)
+    node_factory.join_nodes([l2, l3], wait_for_announce=True)
+    node_factory.join_nodes([l3, l4], wait_for_announce=True)
+    node_factory.join_nodes([l3, l5], announce_channels=False)
+    wait_for(lambda: ['alias' in n for n in l4.rpc.listnodes()['nodes']] == [True, True, True, True])
+
+    offer = l5.rpc.offer(amount='2msat', description='test_offer_with_private_channels_force_issuer_id_multyhop2', force_issuer_id=True)['bolt12']
```

Comment:
assert on the `offer` kind, like you expect that there is a blinded path and offer issuer id, otherwise this test will pass also if there is no offer issuer id because the code should use the blinded path introductory node